Crash: TokenSelector.java line 46 com.alphawallet.app.widget.TokenSelector.<init>

<img width="1148" alt="image" src="https://user-images.githubusercontent.com/735708/188294649-f3765d6f-306d-4926-9c8b-cc0cfd5e0b7f.png">

@justindg gradient problem
`Caused by: android.view.InflateException: Binary XML file line #163: Error inflating class ImageView
                                                           Caused by: android.content.res.Resources$NotFoundException: Drawable io.stormbird.wallet:drawable/select_masking_circle with resource ID #0x7f080192
                                                           Caused by: java.lang.RuntimeException: org.xmlpull.v1.XmlPullParserException: <internal><gradient> tag requires 'gradientRadius' attribute with radial type
                                                           Caused by: org.xmlpull.v1.XmlPullParserException: <internal><gradient> tag requires 'gradientRadius' attribute with radial type`

--- a/app/src/main/res/drawable/select_masking_circle.xml
+++ b/app/src/main/res/drawable/select_masking_circle.xml
@@ -4,8 +4,5 @@
     android:shape="ring"
     android:thicknessRatio="1"
     android:useLevel="false">
-    <gradient
-        android:endColor="?colorSurface"
-        android:gradientRadius="8dp"
-        android:type="radial" />
+    <solid android:color="?colorSurface" />
 </shape>

